### PR TITLE
specify bootstrap version and add header

### LINKED ISSUE #6: Include version in the filename

### LINKED ISSUE #33: Add license header in our JS.

### PR DESCRIPTION
**Pull request by Johann-S**
Fixes #6, fixes #33

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h1 class="mb-0 h5 mr-auto py-1">Bootstrap Customizer</h1>
       <div class="card bg-light text-center">
         <div class="card-body">
           <h5 class="card-title">Customize and download</h5>
-          <p class="card-text">Create a custom <a href="https://getbootstrap.com/" title="Bootstrap website">Bootstrap</a> v4.x build by choosing the components and JavaScript plugins you need.</p>
+          <p class="card-text">Create a custom <a href="https://getbootstrap.com/" title="Bootstrap website">Bootstrap</a> v<span id="bsVersion"></span> build by choosing the components and JavaScript plugins you need.</p>
```
Comment (MartijnCuppens) [inline src/index.html]: Don't really like this. Maybe there is a way to render the number while building?

Comment (Johann-S) [reply src/index.html]: yep, you're right, I'll do that with Webpack !

File: src/js/config.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = Object.freeze({
+  bootstrapVersion: '4.1.3'
```
Comment (XhmikosR) [inline src/js/config.js]: This is something I don't like personally at all. We should have a property in package.json and read it on build time.

Comment (Johann-S) [reply src/js/config.js]: We use it in the build time too 😉 But we need it in source files too

Comment (XhmikosR) [reply src/js/config.js]: Sure, just don't hardcode it.

**Pull request merged**